packages/ra-ui-materialui/src/input/AutocompleteInput.spec.tsx

```diff
@@ -58,6 +58,43 @@ describe('<AutocompleteInput />', () => {
         expect(screen.queryByDisplayValue('foo')).not.toBeNull();
     });
 
+    it('should allow filter to match the selected choice while removing characters in the input', async () => {
```

Comment:
This new test seems fine, but why did you remove the old one testing the emptyValue case? To me we should keep both.

Besides, if my understanding is correct, this test supersedes the `should allow to clear the first character` test, which is useless anyway, so we should remove it IMO.

Reply:
You're actually right!